src/app/content-details/content-details.page.ts

```diff
@@ -207,6 +209,8 @@ export class ContentDetailsPage implements OnInit, OnDestroy {
       this.resumedCourseCardData = extras.resumedCourseCardData;
       this.isSingleContent = extras.isSingleContent;
       this.resultLength = extras.resultsSize;
+      this.hideDownloadShare = (this.cardData.contentData &&
```

Comment:
Add this in extractApiResponse() as well.

Reply:
This changes are done